README.md

```diff
@@ -12,26 +12,37 @@ Flash messages are typically used to display errors on websites that are rendere
 * [Examples](examples/)
 
 ```rust
-use actix_web::{http, server, App, HttpRequest, HttpResponse, Responder};
+use actix_web::{
+    web, 
```

Comment:
There are trailing white spaces on the first entries. Would love those removed.

Reply:
In fact mabye rust format this whole code block, this came out for me:
```
use actix_web::{web, App, HttpRequest, HttpResponse, HttpServer, Responder};
use actix_web_flash::{FlashMessage, FlashMiddleware, FlashResponse};

fn show_flash(flash: FlashMessage<String>) -> impl Responder {
    flash.into_inner()
}

fn set_flash(_req: HttpRequest) -> FlashResponse<HttpResponse, String> {
    FlashResponse::with_redirect("This is the message".to_owned(), "/show_flash")
}

fn main() {
    HttpServer::new(move || {
        App::new()
            .wrap(FlashMiddleware::default())
            .route("/show_flash", web::get().to(show_flash))
            .route("/set_flash", web::get().to(set_flash))
    }).bind("127.0.0.1:8080")
        .unwrap()
        .run()
        .unwrap();
}
```
If you feel strongly about formatting changes I'd be fine with some non standard formatting here in favor of readability, its a README after all.